### ISSUE TITLE
Not working in Netbeans 8.0.2

### ISSUE THREAD

**Issue by og0815** [cismet/beanmill#4]
Hi,

plugin not working in Netbeans 8.0.2. 
Environment: Debian Linux Squeeze, Oracle Java 8 ( 1.8.0_25, 64bit )

Install plain Netbeans (Java SE).
Add update url and install plugin.

After the installation, an Exception happens:
```
java.util.ServiceConfigurationError: javax.xml.parsers.SAXParserFactory: Provider org.apache.xerces.jaxp.SAXParserFactoryImpl not found
	at java.util.ServiceLoader.fail(ServiceLoader.java:239)
	at java.util.ServiceLoader.access$300(ServiceLoader.java:185)
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:372)
	at java.util.ServiceLoader$LazyIterator.access$700(ServiceLoader.java:323)
	at java.util.ServiceLoader$LazyIterator$2.run(ServiceLoader.java:407)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:409)
	at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
	at javax.xml.parsers.FactoryFinder$1.run(FactoryFinder.java:294)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.xml.parsers.FactoryFinder.findServiceProvider(FactoryFinder.java:289)
Caused: java.lang.RuntimeException: Provider for class javax.xml.parsers.SAXParserFactory cannot be created
	at javax.xml.parsers.FactoryFinder.findServiceProvider(FactoryFinder.java:308)
Caused: javax.xml.parsers.FactoryConfigurationError: Provider for class javax.xml.parsers.SAXParserFactory cannot be created
	at javax.xml.parsers.FactoryFinder.findServiceProvider(FactoryFinder.java:311)
	at javax.xml.parsers.FactoryFinder.find(FactoryFinder.java:267)
	at javax.xml.parsers.SAXParserFactory.newInstance(SAXParserFactory.java:127)
	at org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogParser.getUpdateItems(AutoupdateCatalogParser.java:192)
	at org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogProvider.getUpdateItems(AutoupdateCatalogProvider.java:143)
	at org.netbeans.modules.autoupdate.services.UpdateUnitFactory.appendUpdateItems(UpdateUnitFactory.java:173)
	at org.netbeans.modules.autoupdate.services.UpdateUnitFactory.getUpdateUnits(UpdateUnitFactory.java:143)
	at org.netbeans.modules.autoupdate.services.UpdateManagerImpl$Cache.<init>(UpdateManagerImpl.java:271)
	at org.netbeans.modules.autoupdate.services.UpdateManagerImpl.getCache(UpdateManagerImpl.java:231)
	at org.netbeans.modules.autoupdate.services.UpdateManagerImpl.getUpdateUnits(UpdateManagerImpl.java:100)
	at org.netbeans.api.autoupdate.UpdateManager.getUpdateUnits(UpdateManager.java:103)
	at org.netbeans.modules.autoupdate.ui.actions.AutoupdateCheckScheduler.checkUpdateElements(AutoupdateCheckScheduler.java:218)
	at org.netbeans.modules.autoupdate.ui.actions.AutoupdateCheckScheduler$5.run(AutoupdateCheckScheduler.java:189)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1423)
[catch] at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:2033)
``` 

By the way:
On http://cismet.github.io/beanmill/  the update urls are out of date.

**Comment by mscholl** [comment]
Hi, first of all, sry for the late reply, didn't pay too much attention because of holidays...

To the issue: Seems that you're right. I'll see that I can fix it asap.

**Comment by mscholl** [comment]
Hm, sry, my fault, forgot to release the fix since this has already been solved in #3. Please try http://www.cismet.de/updatecenters/beanmill/updates.xml again. Updated gh-pages by the way, too.
